taxi/views.py

```diff
@@ -40,15 +42,60 @@
     queryset = Car.objects.all().select_related("manufacturer")
 
 
-class CarDetailView(LoginRequiredMixin, generic.DetailView):
+class CarDetailView(LoginRequiredMixin, DetailView):
```

Comment:
You don't need to override `template_name` and `context_object_name` if they are the same as default ones. Please, remove these lines if they are not necessary.